docker/ubuntu.dockerfile

```diff
@@ -12,9 +12,9 @@ LABEL maintainer="andrzej.wilczynski@intel.com,dawid.wesierski@intel.com,marek.k
 
 # Install build dependencies and debug tools
 RUN apt-get update -y && \
-    apt-get install -y --no-install-recommends systemtap-sdt-dev && \
     apt-get install -y --no-install-recommends git build-essential meson python3 python3-pyelftools pkg-config libnuma-dev libjson-c-dev libpcap-dev libgtest-dev libsdl2-dev libsdl2-ttf-dev libssl-dev ca-certificates && \
-    apt-get install -y --no-install-recommends m4 clang llvm zlib1g-dev libelf-dev libcap-ng-dev libcap2-bin gcc-multilib && \
+    apt-get install -y --no-install-recommends m4 clang llvm zlib1g-dev libelf-dev libcap-ng-dev libcap2-bin gcc-multilib; \
+    apt-get install -y --no-install-recommends systemtap-sdt-dev || true; \
```

Comment:
why?
this is listed as Debian dependency that is not optional?

Reply:
True - I made some changes before I found out about apt repos updates. Will revert it.

Reply:
Nevertheless Dockerfile had to be refactored and improved. Most of the errors were ignored before refactoring, now this will be more strict.